icarus/src/main/github/App.ts

```diff
@@ -3,7 +3,7 @@ import { UserEventRepository } from "./repositories/UserEventRepository"
 import { WebhookEventService } from "./services/WebhookEventService"
 import { WebhookEndpoint } from "./endpoints/WebhookEndpoint"
 
-const dynamo = new DynamoClient();
+const dynamo = new DynamoClient("");
```

Comment:
Shouldn't be `... new DynamoClient( process.env.TABLE_PREFIX )` ?

Reply:
Ignore my previous comment.
I will change UserEventRepository for using tableprefix and add the prefix consistently